src/main/kotlin/io/github/gradlenexus/publishplugin/internal/StagingRepositoryTransitioner.kt

```diff
@@ -32,14 +32,14 @@ class StagingRepositoryTransitioner(val nexusClient: NexusClient, val retrier: A
 
     //TODO: Add support for autoDrop=false
```

Comment:
The `TODO` comment might probably be removed now :)

Reply:
Might be :).

Nevertheless, we could allow users to disable requesting by the plugin autoDrop in Nexus.